source/refined-github.ts

```diff
@@ -216,3 +216,4 @@ import './features/pr-base-commit.js';
 import './features/unreleased-commits.js';
 import './features/previous-version.js';
 import './features/status-subscription.js';
+import './features/linkify-action.js';
```

Comment:
Probably:

- action-pr-link
- action-back-to-pr

(names should be short)

Reply:
The feature name will be `action-pr-link`

Reply:
We’ve been inconsistent when naming these features. Sometimes we call them “action”, others “workflows”, others “checks”. Wait before renaming until we figure out where this applies.

Reply:
Yes. In this repo, it usually called "action", but I think the formal is "workflow-run".
"workflows" means just recipe file of "workflow-run".
"checks" is more larger concept which can also contains "workflow-run" as "check". The "checks" can be other api jobs not in GitHub.

Reply:
> The "checks" can be other api jobs not in GitHub.

Indeed, those also have a dedicated "checks" page here on GitHub: https://github.com/refined-github/refined-github/issues/1620

But yeah, that page only appears in the PR view, not in the Actions tab.

"action" works well here.

We also have `github-actions-indicators` which is similar but it's yet another slight variation 😅